scene/2d/gpu_particles_2d.cpp

```diff
@@ -912,6 +916,7 @@ GPUParticles2D::GPUParticles2D() {
 	one_shot = false; // Needed so that set_emitting doesn't access uninitialized values
 	set_emitting(true);
 	set_one_shot(false);
+	set_seed(0);
```

Comment:
For consistency with GPUParticles3D's constructor:

```suggestion
	set_fixed_seed(false);
	set_seed(0);
```

Reply:
done :)